packages/fields-content/src/test-fixtures.js

```diff
@@ -0,0 +1,493 @@
+// We don't currently have uniqueness tests for Relationship field types
+import { createItem, getItem, getItems, updateItem } from '@keystonejs/server-side-graphql-client';
+import { Text } from '@keystonejs/fields';
+import { Content } from './index';
+
+export const name = 'Content';
+export { Content as type };
+export const supportsUnique = true;
+
+export const getTestFields = () => {
+  return {
+    name: { type: Text },
+    body: { type: Content },
+  };
+};
+
+const DOC1 =
+  '{"object":"document","data":{},"nodes":[{"object":"block","type":"paragraph","data":{},"nodes":[{"object":"text","text":"This is bold","marks":[{"object":"mark","type":"bold","data":{}}]}]}]}';
+const DOC2 =
+  '{"object":"document","data":{},"nodes":[{"object":"block","type":"paragraph","data":{},"nodes":[{"object":"text","text":"This is BOLD","marks":[{"object":"mark","type":"bold","data":{}}]}]}]}';
+const DOC3 =
+  '{"object":"document","data":{},"nodes":[{"object":"block","type":"paragraph","data":{},"nodes":[{"object":"text","text":"This is a ","marks":[]},{"object":"inline","type":"link","data":{"href":"www.keystonejs.com"},"nodes":[{"object":"text","text":"link","marks":[]}]},{"object":"text","text":" and a ","marks":[]},{"object":"text","text":"Strikethrough.","marks":[{"object":"mark","type":"strikethrough","data":{}}]},{"object":"text","text":" But this is an ","marks":[]},{"object":"text","text":"underline","marks":[{"object":"mark","type":"underline","data":{}}]},{"object":"text","text":".","marks":[]}]},{"object":"block","type":"blockquote","data":{},"nodes":[{"object":"block","type":"paragraph","data":{},"nodes":[{"object":"text","text":"This is a block quote. ","marks":[]}]}]},{"object":"block","type":"paragraph","data":{},"nodes":[{"object":"text","text":"This is italics.","marks":[{"object":"mark","type":"italic","data":{}}]}]}]}';
+
+export const initItems = () => [
+  {
+    name: 'a',
+    body: {
+      disconnectAll: true,
```

Comment:
Do we need the `disconnectAll` in here?

Also, can we fit each of these on a single line, e.g.
```
{ body: { create: { document: DOC1 } } }
```

If prettier allows this it will make this file much easier to read.
```